graphman-client/graphman.configuration.certificate.example

```diff
@@ -0,0 +1,17 @@
+{
+  "sourceGateway": {
+    "address": "https://localhost:8443/graphman",
+    "certificateCertName": "authentication-certificate.pem",
+    "certificateKeyName": "authentication-certificate.key",
```

Comment:
let's standardize the names:
- certFilename
- keyFilename

Reply:
Changed names